core/src/main/scala/org/apache/spark/api/python/PythonRDD.scala

```diff
@@ -330,6 +331,15 @@ private[spark] object PythonRDD extends Logging {
   }
 
   /**
+   * Return an RDD of values from an RDD of (Long, Array[Byte]), with preservePartitions=true
+   *
+   * This is useful for PySpark to have the partitioner after partitionBy()
+   */
+  def valueOfPair(pair: JavaPairRDD[Long, Array[Byte]]): JavaRDD[Array[Byte]] = {
```

Comment:
I think that `JavaPairRDD.values` should do the same thing; is there a reason why we can't call that directly from Python?